make.sh

```diff
@@ -542,9 +550,21 @@ pkg_install_rust() {
 }
 
 pkg_install_web3_deps() {
-    _fold_start "pkg-install-solc"
+    _fold_start "pkg-install-web3-deps"
+    # create and enter virtual environment
+    local python_venv_dir="${PYTHON_VENV_DIR}"
+    _ensure_enter_dir "${python_venv_dir}"
+    python3 -m venv .
```

Comment:
This shouldn't be needed. Can be refined further, and just do the venv creation inside activate.